docs/docs/frequently-asked-questions.md

```diff
@@ -134,4 +134,12 @@ By setting the commission rate ahead of time, they can make sure that they immed
 Yes, by issuing a [`ConsumerModificationProposal`](./features/proposals.md#consumermodificationproposal).
 
 ## Can a Top N consumer chain become Opt-In or vice versa? 
-Yes, by issuing a [`ConsumerModificationProposal`](./features/proposals.md#consumermodificationproposal).
\ No newline at end of file
+Yes, by issuing a [`ConsumerModificationProposal`](./features/proposals.md#consumermodificationproposal).
+
+## How I know when the next validator update will be sent to the consumer chain?
+Validator updates are sent to the consumer chain every `BlocksPerEpoch` blocks.
```

Comment:
Maybe add a note that this does not mean that the changes are immediately shown on the consumer chain due to relayer delays, etc. because (I guess) some people might expect this.

Reply:
Thanks, good idea. Done!